reactioncommerce/reaction
# 1638 Completed Order Logged Out State

Resolves [1638](https://github.com/reactioncommerce/reaction/issues/1638)

**Testing**
- Add Items to your cart and checkout
- On Order Completed page, copy `URL` and paste in incognito browser mode
- Observe that a `notFound` page is presented to user

**Screenshot**
<img width="1680" alt="screen shot 2017-05-03 at 12 06 31 pm" src="https://cloud.githubusercontent.com/assets/10584060/25658102/06cfd95c-2ff9-11e7-8033-3300d3df5a98.png">
